proof/invariant-abstract/X64/ArchCSpace_AI.thy

```diff
@@ -437,12 +452,21 @@ lemma cap_insert_simple_ioports:
          | wpc | simp split del: if_splits)+
   done
 
+lemma cap_insert_simple_ioport_control:
+  "\<lbrace>ioport_control_unique and K (is_simple_cap cap)\<rbrace>
+     cap_insert cap src dest
```

Comment:
are these copy-pasted somehow? we don't indent the function relative to Hoare triple like that anymore?

Reply:
yes, they are based on copy/paste, will fix up the indent